public/index.php

```diff
@@ -1,36 +1,35 @@
 <?php
 
-use Yiisoft\Composer\Config\Builder;
 use Psr\Container\ContainerInterface;
+use Yiisoft\Composer\Config\Builder;
 use Yiisoft\Di\Container;
 use Yiisoft\Http\Method;
 use Yiisoft\Yii\Web\Application;
 use Yiisoft\Yii\Web\SapiEmitter;
 use Yiisoft\Yii\Web\ServerRequestFactory;
 
 require_once dirname(__DIR__) . '/vendor/autoload.php';
-
+require_once dirname(__DIR__) . '/src/globals.php';
```

Comment:
It should not be moved since container won't be available in `globals.php` this way.